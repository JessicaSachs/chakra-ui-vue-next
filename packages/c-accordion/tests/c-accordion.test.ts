import { render } from '@chakra-ui/vue-test-utils'
import CAccordion from '../src'

it('should render properly', () => {
  const { asFragment } = render(CAccordion)
  expect(asFragment()).toMatchSnapshot()
})
