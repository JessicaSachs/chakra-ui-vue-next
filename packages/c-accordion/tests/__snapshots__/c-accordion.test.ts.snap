// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
<DocumentFragment>
  <div
    class="chakra-accordion css-0"
  />
</DocumentFragment>
`;
